third-party/cmake/BuildLibuv.cmake

```diff
@@ -86,12 +81,19 @@ elseif(WIN32 AND MSVC)
   else()
     set(TARGET_CONFIG release)
   endif()
```

Comment:
Is `TARGET_CONFIG` needed anymore?

Reply:
No, and neither is `TARGET_ARCH` so I removed them.